internal/cri/server/sandbox_stats_list.go

```diff
@@ -34,20 +35,33 @@ func (c *criService) ListPodSandboxStats(
 	r *runtime.ListPodSandboxStatsRequest,
 ) (*runtime.ListPodSandboxStatsResponse, error) {
 	sandboxes := c.sandboxesForListPodSandboxStatsRequest(r)
+	stats, errs := make([]*runtime.PodSandboxStats, len(sandboxes)), make([]error, len(sandboxes))
```

Comment:
I think this is true in the existing code too, but: I don't see `errs` read anywhere?  Looks like it's only written, but never read.

Reply:
Isn't `errs` joined together and returned at the end of this RPC call?

Reply:
Yes, thanks.  GitHub hid that line from me...